[TypeScript] [DialogActions] DialogActionsProps missing disableActionSpacing

error TS2339: Property 'disableActionSpacing' does not exist on type 'IntrinsicAttributes & DialogActionsProps & { children?: ReactNode; }'.

- [x] I have searched the [issues](https://github.com/mui-org/material-ui/issues) of this repository and believe that this is not a duplicate.

## Expected Behavior
Code compiles without error.

## Current Behavior
Compile throws error above.

## Steps to Reproduce (for bugs)
``` html
<DialogActions disableActionSpacing>
   ...
</DialogActions
```

## Your Environment
| Tech         | Version |
|--------------|---------|
| Material-UI  | ^1.0.0-beta.37 |
| React        | ^16.2.0 |
| TypeScript   | ^2.7.1 |

--- a/src/Dialog/DialogActions.d.ts
+++ b/src/Dialog/DialogActions.d.ts
@@ -1,8 +1,11 @@
 import * as React from 'react';
+
 import { StandardProps } from '..';
 
 export interface DialogActionsProps
-  extends StandardProps<React.HTMLAttributes<HTMLDivElement>, DialogActionsClassKey> {}
+  extends StandardProps<React.HTMLAttributes<HTMLDivElement>, DialogActionsClassKey> {
+    disableActionSpacing?: boolean;
+  }
 
 export type DialogActionsClassKey = 'root' | 'action';
 

--- a/src/Dialog/DialogActions.d.ts
+++ b/src/Dialog/DialogActions.d.ts
@@ -1,5 +1,4 @@
 import * as React from 'react';
-
 import { StandardProps } from '..';
 
 export interface DialogActionsProps

